addons/hr_holidays/models/hr_leave.py

```diff
@@ -382,7 +382,7 @@ def action_approve(self):
               and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
                 raise UserError(_('You must be %s manager to approve this leave') % (holiday.employee_id.name))
             elif validation_type == 'hr' and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
-                raise UserError(_('You must be a Human Resource Manager to approve this Leave'))
+                raise UserError(_('You must be a Human Resource Manager to approve this leave'))
```

Comment:
==> 'You must be a Human Resource Manager to approve this leave.'

. is missing at the end of the sentence.